cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -790,6 +790,10 @@ public Service generateService() {
         List<Service> services = new ArrayList<>(externalListeners.size());
 
         for (GenericKafkaListener listener : externalListeners)   {
+            if (KafkaListenerType.LOADBALANCER == listener.getType() && listener.getConfiguration() != null && !listener.getConfiguration().getCreateBootstrapService()) {
```

Comment:
MAybe this `if` can have some method in https://github.com/strimzi/strimzi-kafka-operator/blob/main/cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ListenersUtils.java ... you already seem to be using it in 2 places.

Reply:
done.